testing/trino-tests/src/test/java/io/trino/tests/TestLocalEngineOnlyQueries.java

```diff
@@ -59,4 +60,19 @@ public void testResetSession()
     {
         abort("RESET SESSION is not supported by LocalQueryRunner");
     }
+
+    @Test
+    @Override
+    public void testIssue19997()
+    {
+        // TODO Fix LocalQueryRunner execution of functions
+        assertThatThrownBy(super::testIssue19997)
+                .isInstanceOf(AssertionError.class)
+                .hasMessageContaining("[Rows for query [WITH FUNCTION format_topn(input map<varchar, bigint>)")
+                .hasMessageContaining("""
+                        Expecting actual:
+                          (null, null, null)
+                        to contain exactly in any order:
+                          [([AAA, AAA, BBB, BBB, BBB, CCC, CCC, CCC, DDD, DDD], {AAA=2, CCC=3, BBB=3}, CCC=3, BBB=3, AAA=2)]""");
```

Comment:
Apparently LocalQueryRunner and DistributedQueryRunner return different results for this test.

cc @electrum